src/components/TopBar/components/UserMenu/tests/UserMenu.test.tsx

```diff
@@ -12,6 +12,7 @@ describe('<UserMenu />', () => {
     initials: '',
     open: false,
     onToggle: noop,
+    accessibilityLabel: 'User menu',
```

Comment:
This can be removed as we typically only use defaultProps for non-optional additions that have to be included in every test. This also isn't testing anything, so you'll need to add a test, similar to the one for Menu.test.tsx somewhere in this file